src/ProgressOnderwijsUtils/ApprovalTest.cs

```diff
@@ -22,7 +22,7 @@ static string ToApprovalPath(SourceLocation sourceLocation)
             return approvalPath;
         }
 
-        public static ApprovalTest CreateHere([CallerLineNumber] int linenumber = -1, [CallerFilePath] string filepath = null, [CallerMemberName] string membername = null)
+        public static ApprovalTest CreateHere([CallerLineNumber] int linenumber = -1, [CallerFilePath] string filepath = "", [CallerMemberName] string membername = "")

```

Comment:
hmm, is dat beter?  We zouden ook in de SourceLocation constructor `?? throw` kunnen doen.